<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulator</class>
 <widget class="QMainWindow" name="Simulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>961</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSimulation"/>
   <addaction name="separator"/>
   <addaction name="actionStopMotion"/>
   <addaction name="actionReset"/>
   <addaction name="actionGravity"/>
   <addaction name="actionDrag"/>
   <addaction name="actionTerrain"/>
  </widget>
  <action name="actionStopMotion">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/ui/stop.png</normaloff>:/icons/ui/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop Motion</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/ui/reconnect.png</normaloff>:/icons/ui/reconnect.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset Simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionGravity">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/ui/gravity.png</normaloff>:/icons/ui/gravity.png</iconset>
   </property>
   <property name="text">
    <string>Gravity</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionDrag">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/ui/drag.png</normaloff>:/icons/ui/drag.png</iconset>
   </property>
   <property name="text">
    <string>Drag</string>
   </property>
   <property name="toolTip">
    <string>Air Drag</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSimulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/ui/simulator.png</normaloff>:/icons/ui/simulator.png</iconset>
   </property>
   <property name="text">
    <string>Simulation</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionTerrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/ui/ground.png</normaloff>:/icons/ui/ground.png</iconset>
   </property>
   <property name="text">
    <string>Terrain</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
